### PR TITLE
refactor(billing.autorenew): refactor exchange modal to a page

### PR DESCRIPTION
**Pull request by AxelPeter**
- Bump `ovh-ui-angular` to v3.7.0
- Remove component and template files
- Update routing to use directly the component

## Related 
- [ ] https://github.com/ovh-ux/ovh-module-exchange/pull/314

### CODE REVIEW

File: client/app/billing/autoRenew/actions/exchange/exchange-renew.routing.js [hunk 1]
```diff
@@ -1,16 +1,13 @@
...
-    },
-    layout: 'modal',
+    url: '/exchange?organization&exchangeName',
+    component: 'exchangeAccountRenew',
     resolve: {
       goBack: /* @ngInject */ goToAutorenew => goToAutorenew,
       organization: /* @ngInject */ $transition$ => $transition$.params().organization,
-      serviceName: /* @ngInject */ $transition$ => $transition$.params().serviceName,
+      exchangeName: /* @ngInject */ $transition$ => $transition$.params().exchangeName,
+      onError: /* @ngInject */ goBack => result => goBack(result, 'danger'),
```
Comment (marie-j) [inline client/app/billing/autoRenew/actions/exchange/exchange-renew.routing.js]: Maybe it doesn't make sense to use those bindings anymore and use `goBack` directly

File: client/app/billing/autoRenew/autorenew.routing.js [hunk 2]
```diff
@@ -126,7 +126,10 @@ export default /* @ngInject */ ($stateProvider) => {
       sort: /* @ngInject */ $transition$ => $transition$.params().sort,
 
       updateServices: /* @ngInject */ $state => ({ serviceId }) => $state.go('app.account.billing.autorenew.update', { serviceId }),
-      updateExchangeBilling: /* @ngInject */ $state => ({ serviceId }) => $state.go('app.account.billing.autorenew.exchange', { serviceId }),
+      updateExchangeBilling: /* @ngInject */ $state => ({ serviceId }) => {
+        const [organization, exchangeName] = serviceId.split('/service/');
```
Comment (marie-j) [inline client/app/billing/autoRenew/autorenew.routing.js]: `organization === exchangeName` so this trick might not be needed

Comment (AxelPeter) [reply client/app/billing/autoRenew/autorenew.routing.js]: They are the same for now, but is ready if it should changes ;)

File: client/app/billing/autoRenew/actions/exchange/exchange-renew.routing.js (hunk 1 above)
Comment (AxelPeter) [reply client/app/billing/autoRenew/actions/exchange/exchange-renew.routing.js]: Yes, we could use directly `goBack`. I'll update it

File: client/app/billing/autoRenew/autorenew.routing.js (hunk 2 above)
Comment (FredericEspiau) [reply client/app/billing/autoRenew/autorenew.routing.js]: It's been the same for the last decade though 😄

**Comment by ovh-cds** [comment]
CDS Report build-manager#2194.0 ✘  
* Stage 1
  * Build dependencies and inject Sentry ✔ 
* Stage 2
  * Build manager CA ✘  
  * Build manager EU ✘  
  * Build manager US ✘  
  * Tests ✘

**Comment by AxelPeter** [comment]
Yes, I'll add the commit when the release is done.
Thanks :)

**Pull request merged**